packages/mwp-app-render/src/components/uxcapture/UXCaptureEventMark.jsx

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as React from 'react';
+
+type Props = {
+	browserEvent: string,
+	mark: string,
+	children: React$Element<*>,
```

Comment:
Is there a flow type that only allows HTML React elements, or only components that have an `onLoad` prop? We would like to ensure that `onLoad` is handled when injected.

Reply:
Admittedly I had to look this up, but Google says only these tags support `onload`:

```<body>
<iframe>
<img>
<input>
<link>
<script>
<style>```

I'll see if I can make this flow type stricter.

Reply:
ah, boo, so the `<div>` wrapper idea won't work. Okay then - just need to figure out how best to merge our onLoad with an existing onLoad prop